installation/resources/installer-cr-cluster-xip-io.yaml.tpl

```diff
@@ -24,6 +24,8 @@ spec:
       namespace: "kyma-system"
     - name: "nats-streaming"
       namespace: "natss"
+    - name: "event-bus"
+      namespace: "kyma-system"
```

Comment:
Why "event-bus" should be installed before core? I think that "event-bus" should be the first component installed immediately after core

Reply:
I'll move it down after `core` and test